modules/help_editor/test/help_editorTest.php

```diff
@@ -1,42 +1,186 @@
+             'topic'    => 'Test Topic',
+             'content'  => 'This is a test content.',
+             'created'  => '2013-04-05 00:00:00',
+             'updated'  => 'NULL',
+            )
+        );
+    }
+    /**
+     * Delete testing data from database
+     *
+     * @return none
+     */
+    function tearDown()
+    {
+        parent::tearDown();
+        $this->DB->delete("help", array('helpID' => '999999'));
+
+    }
+
     /**
      * Tests that, when loading the help_editor module, some
      * text appears in the body.
      *
+     * @category Test
+     * @package  Loris
+     * @author   Wang Shen <wangshen.mcin@gmail.com>
+     * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+     * @link     https://github.com/aces/Loris
+     *
      * @return void
      */
-    function testHelpEditorDoesPageLoad()
+    function testHelpPageLoad()
     {
         $this->safeGet($this->url . "/help_editor/");
-        $bodyText = $this->webDriver->findElement(WebDriverBy::cssSelector("body"))->getText();
+        $bodyText = $this->webDriver->findElement(WebDriverBy::cssSelector("body"))
+            ->getText();
         $this->assertContains("Help Editor", $bodyText);
-    }
+    }//end test_help_pageload()
     /**
      * Tests that, when loading the help_editor module > edit help submodule, some
      * text appears in the body.
      *
      * @return void
      */
-    function testHelpEditorEditHelpContentDoesPageLoad()
+    function testPageLoad()
     {
-        $this->safeGet($this->url . "/help_editor/edit_help_content/");
-        $bodyText = $this->webDriver->findElement(WebDriverBy::cssSelector("body"))->getText();
-        $this->assertContains("Edit Help Content", $bodyText);
-    }
-}
-?>
\ No newline at end of file
+        $this->safeGet($this->url."/help_editor/edit_help_content/");
+        $assertText = $this->webDriver->findElement(WebDriverBy::cssSelector("body"))
+            ->getText();
+
+        $this->assertContains("Edit Help Content", $assertText);
+
+    }//end test_page_load()
+
+
+    /**
+     * Tests that, when loading the help editor, search Test Topic, click show data,
+     * the Test Data should appear.
+     *
+     * @return void
+     */
+    public function testSearchTopic()
+    {   
+    	try{
+        $this->safeGet($this->url.'/help_editor/');
+        $showdata = $this->webDriver
+            ->findElement(
+                WebDriverBy::Xpath(
+                    "//*[@id='panel-body']".
+                    "/form/div[2]/div/div[1]/input"
+                )
+            );   
+        $searchbox = $this->webDriver
+            ->findElement(WebDriverBy::Name("topic"));
+        $searchbox->sendKeys("Test Topic");
+        $showdata->click();      
+        $assertText = $this->webDriver
+            ->findElement(WebDriverBy::Id("Topic"))->getText();
+        $this->assertContains("Test Topic", $assertText);
+        } catch(Exception $e){
```

Comment:
when does this throw an exception? (And shouldn't the test fail if there is one?)